assistive-webdriver/src/server/request.ts

```diff
@@ -16,28 +16,60 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import request from "request-promise-native";
+import { request } from "http";
```

Comment:
You could have use `node-fetch` which has the nice added value to wrap `request` in an API which mimic browser's native fetch.
With that you would not need to `res.on("data", chunk => { data += chunk; });` 😄

Reply:
Indeed, I will use `node-fetch`. Thank you for your advice!